src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -152,127 +226,126 @@ These variables can be set in your local environment shell before running and th
+*_offline_*
 
-'''''
+The `offline` command saves all the Docker images that Che requires in the `/backup/*.tar` files. Each image is saved as its own file. If the `backup` directory is available on a machine that is disconnected from the Internet and you start Che with the `--offline` command option, the CLI pre-boot sequence loads all the Docker images in the `/backup/` directory.
 
-_restart_
+The `--list` option lists all the core images and optional stack images that can be downloaded. The core system images and the CLI are always saved if an existing `TAR` file is not found. The `--image:<image-name>` command option downloads a single-stack image and can be used multiple times on the command line. You can use the `--all-stacks` option or the `--no-stacks` option to download all or none of the optional stack images.
 
-Performs a `stop` followed by a `start`, respecting `--pull`, `--force`, `--offline`, `--skip:config`, `--skip:preflight`, and `--skip:postflight`.
+*_restart_
 
-'''''
+The `restart` command performs a `stop` action followed by a `start` action, respecting the `--pull`, `--force`, `--offline`, `--skip:config`, `--skip:preflight`, and `--skip:postflight` command options.
 
-_restore_
+*_restore_*
 
-Restores `/instance` to its previous state. You do not need to worry about having the right Docker images. The normal start / stop / restart cycle ensures that the proper Docker images are available or downloaded, if not found.
+The `restore` command restores the `/instance` directory to its previous state. The start/stop/restart cycle ensures that the proper Docker images are available or downloaded, if not found.
 
-This command will destroy your existing `/instance` folder, so use with caution, or set these values to different folders when performing a restore.
+[IMPORTANT]
+====
+We recommend using this command with caution because it deletes the existing `/instance` directory. As a precautionary measure, set these values to different directories when performing a restore action.
+====
 
-'''''
+*_rmi_*
 
-_rmi_
+The `rmi` command deletes the Docker images from the local registry that Che has downloaded for this version.
 
-Deletes the Docker images from the local registry that Che has downloaded for this version.
+*_ssh_*
 
-'''''
+The `ssh` command connects the current terminal where the command is started to the terminal of a machine of the workspace. If no machine is specified in the command, it connects to the default development machine. The syntax is `ssh <workspace-name> [machine-name]`. The ssh connection works only if there is a workspace ssh key setup. A default ssh key is automatically generated when a workspace is created.
```

Comment:
SSH is an acronym and should be all capitals unless referring to the `ssh` command.